### PR TITLE
Updating links and adding a resource for R

### PR DESCRIPTION
**Pull request by olivroy**
## What does this PR do?
Added Tidy modelling with R and updating links to latest versions.

### Why is this valuable (or not)?
tidy models is a new framework to make modelling work

### How do we know it's really free?

The source code is all available on GitHub.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
I don't understand why it fails the alphabetization check. Will need to look more closely - everything looks ok to me

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2130,17 +2130,17 @@ Books on general-purpose programming that don't focus on a specific language are
...
 * [Mastering Software Development in R](https://bookdown.org/rdpeng/RProgDA/) - Roger D. Peng, Sean Kross, and Brooke Anderson 
 * [ModernDive](https://ismayc.github.io/moderndiver-book/) - Chester Ismay, Albert Y. Kim
 * [Practical Regression and Anova using R](http://cran.r-project.org/doc/contrib/Faraway-PRA.pdf) - Julian J. Faraway (PDF)
-* [R for Data Science](http://r4ds.had.co.nz) - Garrett Grolemund, Hadley Wickham
+* [R for Data Science](https://r4ds.hadley.nz) - Hadley Wickham, Mine Çetinkaya-Rundel, and Garrett Grolemund
 * [R for Spatial Analysis](http://www.columbia.edu/~cjd11/charles_dimaggio/DIRE/resources/spatialEpiBook.pdf) (PDF)
 * [R Language for Programmers](http://www.johndcook.com/blog/r_language_for_programmers) - John D. Cook
 * [R Notes for Professionals](https://goalkicker.com/RBook/) - Compiled from StackOverflow Documentation (PDF)
-* [R Packages](http://r-pkgs.had.co.nz) - Hadley Wickham
+* [R Packages]((https://r-pkgs.org) - Hadley Wickham, Jenny Bryan
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2143]: Aha! here's the problem!
```suggestion
* [R Packages](https://r-pkgs.org) - Hadley Wickham, Jenny Bryan
```

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2130,17 +2130,17 @@ Books on general-purpose programming that don't focus on a specific language are
...
 * [Functional Programming](https://dcl-prog.stanford.edu) - Sara Altman, Bill Behrman, Hadley Wickham
-* [Geocomputation with R](https://geocompr.robinlovelace.net) - RobinLovelace, Jakub Nowosad, Jannes Muenchow
+* [Geocomputation with R](https://r.geocompx.org) - Robin Lovelace, Jakub Nowosad, Jannes Muenchow
 * [Introduction to Probability and Statistics Using R](https://github.com/gjkerns/IPSUR) - G. Jay Kerns (PDF)
 * [Learning Statistics with R](https://learningstatisticswithr.com/book/) - Danielle Navarro
 * [Mastering Software Development in R](https://bookdown.org/rdpeng/RProgDA/) - Roger D. Peng, Sean Kross, and Brooke Anderson 
 * [ModernDive](https://ismayc.github.io/moderndiver-book/) - Chester Ismay, Albert Y. Kim
 * [Practical Regression and Anova using R](http://cran.r-project.org/doc/contrib/Faraway-PRA.pdf) - Julian J. Faraway (PDF)
-* [R for Data Science](http://r4ds.had.co.nz) - Garrett Grolemund, Hadley Wickham
+* [R for Data Science](https://r4ds.hadley.nz) - Hadley Wickham, Mine Çetinkaya-Rundel, and Garrett Grolemund
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2139]:
```suggestion
* [R for Data Science](https://r4ds.hadley.nz) - Hadley Wickham, Mine Çetinkaya-Rundel, Garrett Grolemund
```

**Review by eshellman — CHANGES_REQUESTED** [review]
sometimes the linter doesn't point to the real error

**Review by olivroy — COMMENTED** [review]
Thanks.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**